lib/r10k/cli/deploy.rb

```diff
@@ -10,6 +10,15 @@
 
 module R10K::CLI
   module Deploy
+
+    class TransformExcludeSpec
+      def call(input)
+        return true if input == true || input == 'true'
+        return false if input == 'false'
+        raise ArgumentError "Only true/false are valid values not #{input}"
```

Comment:
I'm actually going to remove this error message string because cri swallows those messages and has its own custom messaging when `ArgumentError`s are raise.